scienceapi/settings/common.py

```diff
@@ -82,16 +82,23 @@
     },
 ]
 
-WSGI_APPLICATION = 'scienceapi.wsgi.application'
+WSGI_APPLICATION = 'wsgi.application'
 
 
 # Database
 # https://docs.djangoproject.com/en/1.9/ref/settings/#databases
 
 DATABASES = {
-    'default': env.db("DATABASE_URL", default="postgres:///mozillascience"),
+    'default': {
```

Comment:
I think these changes might be okay since we can probably allow this to be configured.